Readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## PHP Client
 
-### Pre requisits
+### Pre requisites
 1. a working php version > 5.3
 2. composer should be installed, see [composer](http://www.getcomposer.org) for details how to install
```

Comment:
They should.

Anyway we will have proper installation instructions when we have this package in packagist.org

this will include installation directly from composer, or from a git clone.

We might even have videos, with a South African accent, explaining how to install this api